gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,504 @@
+.. license-header
+  SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+  SPDX-License-Identifier: Apache-2.0
+
+############################################################
+Install the NVIDIA GPU DRA Driver and Configure IMEX Support
+############################################################
+
+THe NVIDIA GPU DRA Driver is an additional component you can install after the GPU Operator that enables you to use the Kubernetes DRA feature to define IMEX channel resources that are managed by Kubernetes.
+This page details more information about the GPU DRA Driver, including how to install it and examples of deploying workloads using IMEX channels.
+
+*******************************
+About the NVIDIA GPU DRA Driver

```

Comment:
What is the "GPU DRA Driver" in this context? Is it the driver that is managing the lifecycle of compute domains or access to GPUs themselves?

Reply:
> What is the "GPU DRA Driver" in this context? 

The "NVIDIA GPU DRA Driver" in this document always refers to the component developed in https://github.com/NVIDIA/k8s-dra-driver-gpu

> Is it the driver that is managing the lifecycle of compute domains 

in that sense: yes